src/Summoner/CLI.hs

```diff
@@ -66,10 +67,8 @@ runShow = \case
                     fetchedLicense <- fetchLicense licenseName
                     putTextLn $ unLicense fetchedLicense
   where
-    showGhcVers :: IO ()
-    showGhcVers = mapM_ (infoMessage . T.append "➤ " . showGhcVer) (reverse universe)
-    showLicenses :: IO ()
-    showLicenses = mapM_ (infoMessage . (\ x -> T.append "➤ " $ show (x :: LicenseName))) universe
+    showGhcVers :: IO () = mapM_ (infoMessage . T.append "➤ " . showGhcVer) (reverse universe)
```

Comment:
Better to come back to the previous version with the types on the separate lines 🙂

Reply:
Why is it better?